### ISSUE TITLE
Logger message during training

### ISSUE THREAD

**Issue by huangxt39** [huggingface/transformers#22443]
### System Info

- Platform: Linux-4.15.0-204-generic-x86_64-with-glibc2.27
- Python version: 3.9.13
- Huggingface_hub version: 0.12.1
- PyTorch version (GPU?): 1.13.1 (True)
- Tensorflow version (GPU?): not installed (NA)
- Flax version (CPU?/GPU?/TPU?): not installed (NA)
- Jax version: not installed
- JaxLib version: not installed

### Information

- [ ] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

Hi,

Just to report a problem with the logger. When I use transformers-4.26.1 everything is fine. But after I upgrade to transformers-4.27.3, the logger output in the trainer._inner_training_loop() disappear. In other words, the message below is gone:

***** Running training *****                                                                                                                                   
  Num examples = 53920                                                                                                                                         
  Num Epochs = 5                                                                                                                                               
  Instantaneous batch size per device = 8                                                                                                                      
  Total train batch size (w. parallel, distributed & accumulation) = 16                                                                                        
  Gradient Accumulation steps = 1                                                                                                                              
  Total optimization steps = 16850                                                                                                                             
  Number of trainable parameters = 134302848  

After I downgrade to 4.26.1, the logger works again.

### Expected behavior

-

**Comment by sgugger** [comment]
You need to set the Transformers logging value to this level (unless you do the change of logging before the first import of Transformers), for instance like [this](https://github.com/huggingface/transformers/blob/9b494a1537e3c6a30e5648ab3d4e983380792a91/examples/pytorch/text-classification/run_glue.py#L232)

**Comment by huangxt39** [comment]
I got it. Thanks!
